nsxt/resource_nsxt_policy_lb_pool.go

```diff
@@ -491,17 +501,22 @@ func resourceNsxtPolicyLBPoolRead(d *schema.ResourceData, m interface{}) error {
 		return handleReadError(d, "LBPool", id, err)
 	}
 
+	curState := d.State()
+	if curState.Attributes["active_monitor_path"] != "" {
```

Comment:
We don't expect this attribute to be set in state during the first read (which is the case with create and import).
Deprecation is a bit tricky within the provider since two attributes affectively control the same setting.
I would suggest to check if the user specified the new attribute in intent, and if so, nullify the deprecated one, and vice versa.
If none is specified (potential import scenario), set the new attribute. We can add a note in documentation, stressing out that for new resources (which includes the import use case) only new attribute must be used, otherwise inconsistency may occur.